strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/services/ArtifactManagementServiceImplTest.java

```diff
@@ -359,26 +377,33 @@ public void testForceDelete(@MavenRepository(repositoryId = TFD_RELEASE_WITHOUT_
                    "when allowsForceDeletion is not enabled!");
     }
 
-    @ExtendWith(RepositoryManagementTestExecutionListener.class)
+    @ExtendWith({ RepositoryManagementTestExecutionListener.class,
+                  ArtifactManagementTestExecutionListener.class })
     @Test
-    public void testRemoveTimestampedSnapshots(@MavenRepository(repositoryId = TRTS_SNAPSHOTS,
-                                                                policy = RepositoryPolicyEnum.SNAPSHOT)
-                                               Repository repository)
+    public void testRemoveTimestampedSnapshotsWithCurrentDate(@MavenRepository(repositoryId = TRTS_SNAPSHOTS,
```

Comment:
@ptirador this test case probably better to be named `testRemoveTimestampedSnapshotsByNumberToKeep`. Also in this case we don't need to change snapshots timestamp, because `removeTimestampedSnapshots`  with `numberToKeep` will remove snapshots by number.